manifests/init.pp

```diff
@@ -10,6 +10,7 @@
   Unattended_upgrades::Mail                 $mail                 = {},
   Boolean                                   $minimal_steps        = true,
   Array                                     $origins              = $::unattended_upgrades::params::origins,
+  Array                                     $extra_origins        = [],
```

Comment:
Please enforce the datatype for the array elements. I assume those are all strings?
```suggestion
  Array[String[1]]                          $extra_origins        = [],
```